15-components-lab_v16/src/02-numberguessing.jsx

```diff
@@ -12,6 +12,47 @@
  */
 
 import React from 'react';
+import PropTypes from 'prop-types';
 
-export default React.createClass({
-});
+export default class NumberGuessing extends React.Component {
+
+    static propTypes = {
+        num: PropTypes.number,
+        rand: PropTypes.number
+      };
+    
```

Comment:
Why did you decide to store `rand` and `num` as properties but `message` and `guess` as state?